schemas/2014-04-01-preview/deploymentTemplate.json

```diff
@@ -40,9 +40,7 @@
               },
               {
                 "oneOf": [
-                  { "$ref": "https://schema.management.azure.com/schemas/2019-09-16/Microsoft.HealthcareApis.json#/resourceDefinitions/services" },
                   { "$ref": "https://schema.management.azure.com/schemas/2019-02-01-preview/Microsoft.AppConfiguration.json#/resourceDefinitions/configurationStores" },
-                  { "$ref": "https://schema.management.azure.com/schemas/2018-08-20-preview/Microsoft.HealthcareApis.json#/resourceDefinitions/services" },
```

Comment:
This file and `schemas/2015-01-01/deploymentTemplate.json` are in maintenance mode and won't pick up new types or autogenerated schemas. So I think it's best to leave this as-is (don't remove anything, but don't add the new version)

Reply:
Ok I will go revert this file change and the 2015-01-01 template as well. I was going based on the below steps: https://github.com/Azure/azure-resource-manager-schemas/blob/master/generator/README.md#onboarding-teams-to-autogeneration-pipeline